src/scripts/test_podcast.py

```diff
@@ -2,15 +2,18 @@
 import json
 
 SESSION_TOKEN = 0
-SERVER_URL = "http://SERVER_URL/api/v1/search/users/A/?offset=0&max=50"
+SERVER_URL = 'localhost:5000'
```

Comment:
will we always be on port 5000?

Reply:
Just a dummy value. We can put in the real destination in case we ever want to test against production